src/status_im2/common/floating_button_page/view.cljs

```diff
@@ -0,0 +1,113 @@
+(ns status-im2.common.floating-button-page.view
```

Comment:
I added this helper component which takes the logic from create profile page and makes it into a common component. I.e a page with a button that floats at the bottom and has a container in it.

wallet save address (this is just the design file)
<img width="233" alt="Screenshot 2023-10-18 at 09 29 44" src="https://github.com/status-im/status-mobile/assets/22799766/b71fa8c3-d288-42be-ad03-e57f95ea93c8">
onboarding create profile (this is just the design file)
<img width="233" alt="Screenshot 2023-10-18 at 09 30 36" src="https://github.com/status-im/status-mobile/assets/22799766/54a3da17-a913-49e4-8a42-467680ff527f">

Reply:
@jo-mut maybe you can lmk what you think about this common component too 👍

Reply:
@J-Son89  looks good. I think its good we have it. There are several screens simply have this layout. Nice one

Reply:
@J-Son89 I think we have an issue to refactor several screens to use this component

Reply:
Yeah, I think that might be the case too. Need to make sure we cover it well with unit tests etc too for that reason